x-pack/plugins/observability/public/pages/alert_details/components/alert_summary.tsx

```diff
@@ -5,173 +5,151 @@
  * 2.0.
  */
 import React from 'react';
-import {
-  EuiText,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiSpacer,
-  EuiPanel,
-  EuiTitle,
-  EuiLink,
-  EuiToolTip,
-  EuiIcon,
-  EuiButtonEmpty,
-} from '@elastic/eui';
+import { EuiText, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
+import moment from 'moment';
+import { ALERT_STATUS_ACTIVE, ALERT_STATUS_RECOVERED } from '@kbn/rule-data-utils';
+import { asDuration } from '../../../../common/utils/formatters';
 import { PageHeaderProps } from '../types';
 import { useKibana } from '../../../utils/kibana_react';
 import { AlertStatusIndicator } from '../../../components/shared/alert_status_indicator';
+import { DEFAULT_DATE_FORMAT } from '../constants';
 
 export function AlertSummary({ alert }: PageHeaderProps) {
```

Comment:
Shall we use `AlertSummaryProps` instead of `PageHeaderProps`?

Reply:
yes indeed `AlertSummaryProps` makes more sense, will change this.